tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -916,3 +916,94 @@ def process(self, image, annotation_meta=None):
         image.metadata.update({'multi_infer': True, 'scales': scales})
 
         return image
+
+
+class CenterNetAffineTransformation(Preprocessor):
+    __provider__ = 'centernet_affine_transform'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'dst_width': NumberField(
+                value_type=int, optional=False, min_value=1, description="Destination width for image."
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=False, min_value=1, description="Destination height for image."
+            ),
+            'scale': NumberField(
+                value_type=int, optional=True, default=1,
+                description="Scale for input image"
+            )
+        })
+
+        return parameters
+
+    def configure(self):
+        self.input_height, self.input_width = get_size_from_config(self.config)
```

Comment:
get size from config also assume that `size` parameter also supported for case when dst_height and dts_width are equal